drivers/azure/azureutil/azureutil.go

```diff
@@ -470,8 +470,7 @@ func (a AzureClient) DeleteVirtualMachineIfExists(ctx context.Context, resourceG
 	if vmProperties := vm.VirtualMachineProperties; vmProperties != nil {
 		// TODO: remove unattached managed disk, requires azure sdk upgrade
```

Comment:
Is this TODO still required?

Reply:
good catch, removed in the latest commit